bftengine/CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ if(${BUILD_COMM_TCP_PLAIN})
     set(corebft_source_files ${corebft_source_files} src/communication/PlainTcpCommunication.cpp)
 endif()
 if(${BUILD_COMM_TCP_TLS})
-    set(corebft_source_files ${corebft_source_files} src/communication/TlsTCPCommunication.cpp)
+    set(corebft_source_files ${corebft_source_files} src/communication/TlsTCPCommunication.cpp tests/testMsgsCertificate/msgsCertificate_test.cpp)
```

Comment:
Why is this necessary? And if it is, why only in the case when ${BUILD_COMM_TCP_TLS} is on ?